### PR TITLE
DM-35663: add a mock DIMM

### PR DESCRIPTION
**Pull request by r-owen**
and simplify the doc layout

### CODE REVIEW

File: python/lsst/ts/dimm/controllers/astelco_dimm.py [hunk 1]
```diff
@@ -109,40 +159,59 @@ def __init__(self, log):
         self.reader = None
         self.writer = None
 
-        self.cmd_lock = asyncio.Lock()
-        self.cmd_list = {}
+        # Lock to prevent commands from being sent until
+        # until authorization is complete.
```
Comment (tribeiro) [inline python/lsst/ts/dimm/controllers/astelco_dimm.py:177]: word "until" is duplicated

**Review by tribeiro — COMMENTED** [review]
These are some pretty impressive updates! Thank you very much! I gave a first pass on the code and only have one small comment about a duplicated string in a comment. 

Do you want to hold on merging this PR until we are able to test it or you rather merge it now and open a new PR if we need updates after testing?

**Comment by r-owen** [comment]
I would prefer to test the code before merging this PR

**Review by tribeiro — APPROVED** [review]
Looks good, I'm glad we waited to test it before merging as we ended up finding additional things to fix. Please, remember to reorganize the commit before merging. 

I pushed updates to the conda recipe.

**Pull request merged**